src/ImageRunModal.jsx

```diff
@@ -687,7 +687,44 @@ export class ImageRunModal extends React.Component {
                 <Tabs activeKey={activeTabKey} onSelect={this.handleTabClick}>
                     <Tab eventKey={0} title={<TabTitleText>{_("Details")}</TabTitleText>} className="pf-c-form pf-m-horizontal">
                         { this.props.userServiceAvailable && this.props.systemServiceAvailable &&
-                        <FormGroup isInline hasNoPaddingTop fieldId='run-image-dialog-owner' label={_("Owner")}>
+                        <FormGroup isInline hasNoPaddingTop fieldId='run-image-dialog-owner' label={_("Owner")}
+                                   labelIcon={
+                                       <Popover aria-label={_("Owner help")}
+                                          enableFlip
+                                          bodyContent={
+                                              <>
+                                                  <TextContent>
+                                                      <Text component={TextVariants.h4}>{_("System")}</Text>
+                                                      <TextList>
+                                                          <TextListItem>
+                                                              {_("Ideal for running services")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Resource limits can be set")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Checkpoint/restore support")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Ports under 1024 can be forwarded")}
+                                                          </TextListItem>
+                                                      </TextList>
+                                                  </TextContent>
+                                                  <TextContent>
+                                                      <Text component={TextVariants.h4}>{cockpit.format("$0 $1", _("User:"), this.props.user)}</Text>
+                                                      <TextList>
+                                                          <TextListItem>
+                                                              {_("Ideal for development")}
+                                                          </TextListItem>
```

Comment:
@garrett I can't think of more points for the user session, can you?

Reply:
Looks good. I think these points work well.

Are there any network considerations besides the ports?

Out of scope, but interesting and distantly related: 

(The low ports is such is an absurd default policy in this day and age, but that's a different topic and way out of scope — and I think the policy can even be changed manually, but we might even hardcode that somewhere, either in Podman or Cockpit-Podman).

And, someone could remap the port using firewall rules too. (This would actually be a neat feature for Cockpit to handle, probably in the Firewall page, and it would be awesome if we had integration for both Cockpit-Podman and Cockpit-Machines.) But, again, this is out of scope from this PR.

Reply:
They are also ideal (or sometimes even required, with e.g. encrypted home dirs) for working with your $HOME data, like handing over your ~/Photos dir as a volume to your image gallery container.